### PR TITLE
[Windows] Add azcopy utility

### PR DESCRIPTION
**Pull request by andy-mishechkin**
# Description
New tool **azcopy10** deployment has been added.
## Check list
- [X] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Validate-AzCopy.ps1 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+if (Get-Command -Name 'azcopy') {
+    Write-Host "azcopy $(azcopy --version) on path"
+}
+else {
```
Comment (vmapetr) [inline images/win/scripts/Installers/Validate-AzCopy.ps1:4]: nit: please follow general code style, and place braces and conditions at the same line - `} else {`

File: images/win/scripts/Installers/Validate-AzCopy.ps1 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+if (Get-Command -Name 'azcopy') {
+    Write-Host "azcopy $(azcopy --version) on path"
```
Comment (al-cheb) [inline images/win/scripts/Installers/Validate-AzCopy.ps1:2]: We should remove duplicated azcopy word:
```
PS > "azcopy $(azcopy --version) on path"
azcopy azcopy version 10.4.3 on path
```

**Comment by al-cheb** [comment]
Shouldn't we update packer json configs?

File: images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1 [hunk 3]
```diff
@@ -141,6 +141,10 @@ function Get-AzureCLIVersion {
     return "Azure CLI $azureCLIVersion"
 }
 
+function Get-AzCopyVersion {
+    return ($(azcopy --version) -replace "version ")
```
Comment (al-cheb) [inline images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1:145]: nitpick: Do we really need return and brackets?  `(azcopy --version).replace('version ','')`

File: images/win/scripts/Installers/Validate-AzCopy.ps1 (hunk 2 above)
Comment (andy-mishechkin) [reply images/win/scripts/Installers/Validate-AzCopy.ps1]: Fixed

File: images/win/scripts/Installers/Validate-AzCopy.ps1 (hunk 1 above)
Comment (andy-mishechkin) [reply images/win/scripts/Installers/Validate-AzCopy.ps1]: Done

File: images/win/scripts/Installers/Install-AzCopy.ps1 [hunk 4]
```diff
@@ -0,0 +1 @@
+Choco-Install -PackageName azcopy10
```
Comment (vmapetr) [inline images/win/scripts/Installers/Install-AzCopy.ps1:1]: Nitpick: Please add header. Something like
```
################################################################################
##  File:  Install-AzCopy.ps1
##  Desc:  Install AzCopy10
################################################################################
```

File: images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1 (hunk 3 above)
Comment (andy-mishechkin) [reply images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1:145]: "return" is present in all functions in SoftwareReport.Tools.psm1 module.
I've decided to use the same style

File: images/win/scripts/Installers/Install-AzCopy.ps1 (hunk 4 above)
Comment (andy-mishechkin) [reply images/win/scripts/Installers/Install-AzCopy.ps1:1]: Done

**Comment by vmapetr** [comment]
BTW, as I see, [azcopy](https://chocolatey.org/packages/azcopy) and [azcopy10](https://chocolatey.org/packages/azcopy10) are different packages on chocolately. 
```
It (azcopy10) has a whole new syntax incompatible with the old v8 versions. You can use them side-by-side.
```
Perhaps we should install azcopy as well?

**Comment by andy-mishechkin** [comment]
> Shouldn't we update packer json configs?

Done

**Comment by andy-mishechkin** [comment]
According to https://github.com/actions/virtual-environments/issues/1004:
- Desired version: v10

**Comment by andy-mishechkin** [comment]
> BTW, as I see, [azcopy](https://chocolatey.org/packages/azcopy) and [azcopy10](https://chocolatey.org/packages/azcopy10) are different packages on chocolately.
> 
> ```
> It (azcopy10) has a whole new syntax incompatible with the old v8 versions. You can use them side-by-side.
> ```
> 
> Perhaps we should install [azcopy(8)](https://chocolatey.org/packages/azcopy) as well?
According to #1004:

Desired version: v10

**Pull request merged**